sofl/preferences.py

```
@@ -473,3 +481,39 @@ def on_response(widget: Any, response: str) -> None:
         # Set the source row subtitles
         self.resolve_locations(source)
         self.update_source_action_row_paths(source)
+    def setup_online_fix_settings(self) -> None:
+        """Setup parameters for Online-Fix"""
+        # Check for the key in settings
+        try:
+            # Try to get the value if the key exists
+            current_path = shared.schema.get_string("online-fix-install-path")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)